src/common/sentry.ts

```diff
@@ -21,95 +21,78 @@
 
 import { CaptureConsole, Dedupe, Offline } from "@sentry/integrations";
 import type { Event as SentryEvent } from "@sentry/types";
-import { sentryDsn, sentryDsnIsValid, isProduction } from "./vars";
+import { sentryDsn, isProduction } from "./vars";
 import { UserStore } from "./user-store";
 import logger from "../main/logger";
 
 // https://www.electronjs.org/docs/api/process#processtype-readonly
-const electronProcessType = ["browser", "renderer", "worker"] as const;
+type ElectronProcessType = "browser" | "renderer" | "worker";
+
+const processType = process.type as ElectronProcessType;
 
 export const integrations = [
   new CaptureConsole({ levels: ["error"] }),
   new Dedupe(),
   new Offline()
 ];
 
-const initialScope = (processType: typeof electronProcessType[number]) => {
-  return {
-    tags: {
-      // "translate" browser to 'main' as Lens developer more familiar with the term 'main'
-      "process": processType === "browser" ? "main" : "renderer"
-    }
-  };
+const initialScope = {
+  tags: {
+    // "translate" browser to 'main' as Lens developer more familiar with the term 'main'
+    "process": processType === "browser" ? "main" : "renderer"
+  }
 };
 
-export const environment = isProduction ? "production" : "development";
+const environment = isProduction ? "production" : "development";
 
-const logInitError = (reason: string) => {
+function logInitError(reason: string) {
   logger.warn(`⚠️  [SENTRY-INIT]: ${reason}, Sentry is not initialized.`);
-};
```

Comment:
I wouldn't change these unless we have linter rule to force using function over const.

Reply:
Okay I can revert. We should have a discussion about some more lint rules